htsohm/mutate.py

```diff
@@ -157,111 +155,125 @@ def write_children_definition_files(run_id, generation):
     wd = os.environ['HTSOHM_DIR']
     strength_array_file = os.path.join(wd, 'config', run_id + '.npy')
     strength_array = np.load(strength_array_file)
-    children = session.query(Material).filter(Material.run_id == run_id,
-        Material.generation == generation).all()
 
-    for child in children:
-        child_id = child.id
-        ########################################################################
-        # write force_field.def
-        child_forcefield_directory = os.path.join(fd, run_id + '-' + str(child_id))
-        os.mkdir(child_forcefield_directory)
-        force_field_file = os.path.join(child_forcefield_directory, 'force_field.def')
-        write_force_field(force_field_file)                        # WRITE FORCE_FIELD.DEF
+    ########################################################################
+    # add row to database
+    new_material = Material(run_id, generation, 'none')
+    session.add(new_material)
+    session.flush()
```

Comment:
the flush() here is preempting your session.commit() one level up. Does it still work as expected?